simplq/src/store/asyncActions/index.js

```diff
@@ -10,6 +10,7 @@ export { deleteQueue, useDeleteQueue } from './deleteQueue';
 export { getQueueStatus, useGetQueueStatus } from './getQueueStatus';
 export { getQueueStatusByName, useGetQueueStatusByName } from './getQueueStatusByName';
 export { joinQueue, useJoinQueue } from './joinQueue';
+export { getActiveTokens, useGetActiveTokens } from './getActiveTokens';
 
 export { getToken, useGetToken } from './getToken';
```

Comment:
Suggestions on better names?

Reply:
`getQueueTokens({ queueId })` and `useGetQueueTokens()`

```js
const getQueueTokens = useGetQueueTokens()

dispatch(getQueueTokens({ queueId }))
```